src/pages/groupComparison/GroupComparisonUtils.tsx

```diff
@@ -623,4 +623,28 @@ export function convertPatientsStudiesAttrToSamples(
             })
         )
     }));
+}
+
+export function partitionCases(
```

Comment:
can we have a more descriptive name or a comment on this function that indicates the nature of the partition?